form.yml

```diff
@@ -60,12 +72,82 @@ cluster:
       $type: boolean
       $default: false
       $optional: true
-    device:
-      $name: SBD device url
-      $type: text
-      $placeholder: /dev/vdb
+    device_list:
+      $visibleIf: .diskless_checkbox == false
+      $name: SBD Device(s)
       $optional: true
-
+      $type: edit-group
+      $itemName: ""
+      $minItems: 0
+      $prototype:
+        $name: SBD device url
+        $optional: true
+        $type: text
+    configure_sbd_checkbox:
+      $name: Configure sbd cluster resource
+      $type: boolean
+      $default: false
+      $help: This updates the resource stonith-sbd created if sbd is enabled. sbd configuration will be overwritten if the resource is defined in the configure template
+    configure_resource:
+      $name: Configure sbd resource
+      $visibleIf: .configure_sbd_checkbox == true
+      $optional: true
+      $type: group
+      force_configure_checkbox:
+        $name: Force commit
+        $type: boolean
+        $default: false
+        $help: Force commit. Use with caution, this can allow invalid parameters in the configuration
+      configure_resource_params:
+        $name: Configure resource params
+        $type: boolean
+        $default: false
+        $help: Mark this option to configure sbd resource parameters
+      resource_params:
```

Comment:
We need to change this. `pcmk_delay_max` is only and example. This just should be a list of `key/value` entries